app.py

```diff
@@ -73,12 +76,12 @@
 )
 
 
-def _buildlog_metadata(api_endpoint: str = None, build_log: str = None) -> dict:
+def _buildlog_metadata(api_endpoint: str = Optional[None], build_log: str = Optional[None]) -> Dict[str, str]:
```

Comment:
shouldn't these be `foo: Optional[str] = None`?

Reply:
Otherwise their default value will be set to a type annotation

Reply:
Yes, silly mistake (tired eyes? :)) Created typing fix for this typing fix in 6f04ae8, PTAL